cassdegrees/templates/registration/password_change_done.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load breadcrumbs %}
+
+{% block page-title %}Change Password{% endblock %}
+
+{% block subtitle %}CASS Staff Password Change{% endblock %}
+
+{% block breadcrumb-trail %}
+    {% finalcrumb 'Change Password' %}
+{% endblock %}
+
+{% block content %}
+    <div class="msg-success">Your password has been successfully changed</div>
+    <br />
+    <a href="/">Back to Hompage</a>
```

Comment:
Homepage

Reply:
Not sure what you are requesting here.

Reply:
Typo - you called it `hompage`. not `homepage`.

Reply:
Oh I see it now, thanks :')